### PR TITLE
implement "center messages" option

### PR DESCRIPTION
**Pull request by rfomin**
Feature request: [[doomworld]](https://www.doomworld.com/forum/topic/112333-this-is-woof-900-feb-24-2022-updated-winmbf/?do=findComment&comment=2496984)

While we are here, can we remove "message review" mode? I accidentally turned it on one day and had to refer to the source code to turn it off. This is a feature from Boom, but I don't think anyone has ever used it. Also I still don't know how to enable it in PrBoom+.

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
> While we are here, can we remove "message review" mode? 

Yes, kill it.

File: Source/hu_stuff.c [hunk 1]
```diff
@@ -1700,6 +1703,11 @@ void HU_Ticker(void)
   if ((showMessages || message_dontfuckwithme) && plr->message &&
       (!message_nottobefuckedwith || message_dontfuckwithme))
     {
+      if (message_centered)
```
Comment (fabiangreffrath) [inline Source/hu_stuff.c:1706]: If you hit Enter to display a previous message, will this also appear properly centered?

**Comment by fabiangreffrath** [comment]
> Yes, kill it.

Also, please remove the corresponding embedded patches from info.c.

Should we move the "Secret Revealed message" menu item to the Messages menu? There'll be a lot of vacant space after these changes.

File: Source/hu_stuff.c (hunk 1 above)
Comment (rfomin) [reply Source/hu_stuff.c:1706]: Yes, it works.

**Comment by rfomin** [comment]
> Should we move the "Secret Revealed message" menu item to the Messages menu? There'll be a lot of vacant space after these changes.

Ok. I think we can leave the option for multiple lines, but remove the background and the "message review".

**Comment by fabiangreffrath** [comment]
Is this ready?

**Comment by rfomin** [comment]
Yes, it's ready.

**Pull request merged**